lib/email_alert_signup_api.rb

```diff
@@ -38,7 +38,7 @@ def subscriber_list_options
     options = { "title" => subscriber_list_title }
     options["combine_mode"] = combine_mode if combine_mode
     if facet_groups?
-      options["links"] = facet_groups
+      options["links"] = tags
```

Comment:
This seems an odd if-else block now (both set to `tags`) - could it be refactored?